File: Iniciante/URI 1044.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var line = lines.shift().split(" ");
var A = parseInt(line[0]);
var B = parseInt(line[1]);

if (A % B == 0 || B % A == 0) 
    console.log('Sao Multiplos');
else 
    console.log('Nao sao Multiplos');

File: Iniciante/URI 1044.dart
import 'dart:io';

void main() {
  List<String> input = stdin.readLineSync().split(" ");
  int a = int.parse(input[0]);
  int b = int.parse(input[1]);
  if (b % a != 0 && a % b != 0) {
    print("Nao sao Multiplos");
  } else {
    print("Sao Multiplos");
  }
}
